### PR TITLE
docs: add table options to table-management

### LINKED ISSUE #229: Update docs for feat: support table ttl
A document change request is generated from
https://github.com/GreptimeTeam/greptimedb/pull/1052

### PR DESCRIPTION
**Pull request by v0y4g3r**
- Add table option reference to "Table Management" sections
- Add `write_buffer_size` table option docs.

Fix #229

### CODE REVIEW

File: docs/reference/sql/create.md [hunk 1]
```diff
@@ -66,8 +66,9 @@ Users can add table options by using `WITH`. The valid options contain the follo
 
 | Option  | Description  | Value |
 |---|---|---|
-| ttl  | The storage time of the table data  |   String value, such as `'60m'`, `'1h'` for one hour, `'14d'` for 14 days etc. |
-|  regions | The region number of the table  | Integral value, such as 1, 5, 10 etc. |
+| `ttl`  | The storage time of the table data  |   String value, such as `'60m'`, `'1h'` for one hour, `'14d'` for 14 days etc. |
+|  `regions` | The region number of the table  | Integral value, such as 1, 5, 10 etc. |
+| `write_buffer_size` | Memtable size of the table | String value representing a valid size, such as `32MB`, `128MB`, etc. |
```
Comment (killme2008) [inline docs/reference/sql/create.md:71]: What's the default value of `write_buffer_size`?

Comment (v0y4g3r) [reply docs/reference/sql/create.md]: > What's the default value of `write_buffer_size`?

Fixed

File: docs/reference/sql/create.md [hunk 2]
```diff
@@ -66,8 +66,9 @@ Users can add table options by using `WITH`. The valid options contain the follo
 
 | Option  | Description  | Value |
 |---|---|---|
-| ttl  | The storage time of the table data  |   String value, such as `'60m'`, `'1h'` for one hour, `'14d'` for 14 days etc. |
-|  regions | The region number of the table  | Integral value, such as 1, 5, 10 etc. |
+| `ttl`  | The storage time of the table data  |   String value, such as `'60m'`, `'1h'` for one hour, `'14d'` for 14 days etc. |
```
Comment (xtang) [inline docs/reference/sql/create.md:69]: Is it possible to list all the time units here? For example,
ms - millisecond,
s - second,
m - minute,
h - hour,
d - day,
w - week,
y - year

**Review by xtang — APPROVED** [review]
LGTM

**Pull request merged**